CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ Change Log
 Version 2.2.0 *(2021-XX-XX)*
 -----------------------------
 
-* 
+* Fix: Files generated with `Unknown` prefix even being known in Windows OS
```

Comment:
Instead of explaining what's the fix about, explain what does it solve. In this case: Compilation error when compiling on windows